internal/btf/ext_info.go

```diff
@@ -328,6 +328,26 @@ type LineInfo struct {
 	LineCol     uint32
 }
 
+// FileName returns the file name as string
+func (li LineInfo) FileName(btfSpec *Spec) (string, error) {
+	return btfSpec.strings.Lookup(li.FileNameOff)
+}
+
+// Line returns the line of source code as a string
+func (li LineInfo) Line(btfSpec *Spec) (string, error) {
+	return btfSpec.strings.Lookup(li.LineOff)
+}
+
+// LineNum returns the line number within the FileName
+func (li LineInfo) LineNum() uint32 {
+	return li.LineCol >> 10
```

Comment:
Why 10 bits to the right?

Reply:
That is the amount of `1`s in `0x3ff`. LineCol is has 22 bits of line number and 10 bits of column number packed into one value. Perhaps a link to the kernels BTF docs can help. But I think this is moot since #572 will change the LineInfo to just have separate Line and Col numbers exported which replaces this.